lib/mojito.js

```diff
@@ -506,11 +525,19 @@ Mojito = (function ()
                 return val;
             },
             /**
-             * get random value from different engine
+             * get random value from decisionAdapter
              */
             getRandom: function()
             {
-                return this.getSeededRNGRandom();
+                if (!('decisionCnt' in this.options))
```

Comment:
Sorry to change my mind here - could you please rename `decisionCnt` to `decisionIdx`? 

`Cnt` is a little too close to a bad word in English and the more I look at it, the more I want to distance our code from it.

Reply:
@kingo55 
You're right, I will change it to ```decisionIdx```.

Reply:
Fixed.